poe-tasks/get-modified-connectors.sh

```diff
@@ -44,10 +48,24 @@ ignore_patterns=(
 ignore_globs="($(IFS='|'; echo "${ignore_patterns[*]}"))$"
 
 # 3) collect all file changes
-committed=$(git diff --name-only origin/"$DEFAULT_BRANCH"...HEAD)
-staged=$(git diff --cached --name-only)
-unstaged=$(git diff --name-only)
-untracked=$(git ls-files --others --exclude-standard)
+#committed=$(git diff --name-only origin/"$DEFAULT_BRANCH"...HEAD)
+#staged=$(git diff --cached --name-only)
+#unstaged=$(git diff --name-only)
+#untracked=$(git ls-files --others --exclude-standard)
```

Comment:
I think we can fully delete these, since they are moved below to the 'else' block.

```suggestion
```

Reply:
Good catch. I left them there because I was just testing.